pkg/quota/evaluator/core/pods.go

```diff
@@ -244,13 +245,13 @@ func podComputeUsageHelper(requests api.ResourceList, limits api.ResourceList) a
 		result[api.ResourceLimitsEphemeralStorage] = limit
 	}
 	for resource, request := range requests {
-		if quota.ContainsPrefix(requestedResourcePrefixes, resource) {
+		if quota.ContainsPrefix(requestedResourcePrefixes, resource) || helper.IsExtendedResourceName(resource) {
```

Comment:
This revision is kind of hijacking the hugepages logic for extended resources, although the call to maskResourceWithPrefix below is unnecessary. So I'd suggest we add two separate loops for extended resources with comments.
The current logic is unnecessarily coupling extended resources to hugepages. This is not good for future maintenance.

Reply:
SGTM, done.